src/glamor.re

```diff
@@ -734,4 +734,4 @@ let stroke = v => Property("stroke", v);
 
 let strokeWidth = v => Property("strokeWidth", v);
 
-let strokeMiterlimit = v => Property("strokeMiterlimit", v);
+let strokeMiterlimit = v => Property("strokeMiterlimit", v);
```

Comment:
Is this intentional? I'd rather keep the newline at the end of the file.

Reply:
no it isn't, that must have been from auto format from merlin.